### PR TITLE
Set initial joint configurations to mean of joint limits

### PR DESCRIPTION
**Pull request by domire8**
I added a function that sets the initial joint configuration to the mean of the joint position limits after the robot is loaded. The rest of the diff is just auto-formatting.

I'd like to bring this in such that the ros demo in control_libraries is easier to explain and use.

As we discussed, it would be nice to create a service for this, such that the user can just restart his control node without restarting the main simulation.

### CODE REVIEW

File: ros/src/pybullet_ros/pybullet_ros.py [hunk 1]
```diff
@@ -100,11 +105,12 @@ def handle_reset_simulation(self, req):
 
     def start_gui(self, gui=True):
         """start physics engine (client) with or without gui"""
-        if(gui):
+        if (gui):
```
Comment (eeberhard) [inline ros/src/pybullet_ros/pybullet_ros.py]:
```suggestion
        if gui:
```

File: ros/src/pybullet_ros/pybullet_ros.py [hunk 2]
```diff
@@ -159,12 +168,20 @@ def init_pybullet_robot(self):
...
-                                           useFixedBase=fixed_base, flags=urdf_flags)
+                                baseOrientation=robot_spawn_orientation,
+                                useFixedBase=fixed_base, flags=urdf_flags)
+
+    def set_initial_joint_configuration(self, robot, joint_dict):
+        """Function to set initial joint configuration to the mean of the joint position limits."""
+        for joint_id, joint_name in joint_dict.items():
+            lower_lim = self.pb.getJointInfo(robot, joint_id)[8]
+            upper_lim = self.pb.getJointInfo(robot, joint_id)[9]
+            print(-(upper_lim - lower_lim) / 2)
```
Comment (eeberhard) [inline ros/src/pybullet_ros/pybullet_ros.py]: is this print needed?

**Review by eeberhard — APPROVED** [review]
It's simple enough and looks good to me. Since there is already some reformatting / refactoring of the file I added a few minor comments.

File: ros/src/pybullet_ros/pybullet_ros.py (hunk 2 above)
Comment (domire8) [reply ros/src/pybullet_ros/pybullet_ros.py]: definitely not thanks

**Pull request merged**